clang/lib/CodeGen/TargetInfo.h

```diff
@@ -413,6 +414,12 @@ class TargetCodeGenInfo {
     return nullptr;
   }
 
+  void setFnAttributes(const TargetInfo::BranchProtectionInfo &BPI,
```

Comment:
```suggestion
  void setBranchProtectionFnAttributes(const TargetInfo::BranchProtectionInfo &BPI,
```
Or so? The current name seems a bit too generic.

Reply:
agree, added.